mangadex-dl.py

```diff
@@ -132,12 +135,17 @@ def dl(manga_id, lang_code, zip_up, tld="org"):
 			dest_filename = pad_filename(filename)
 			outfile = os.path.join(dest_folder, dest_filename)
 
-			r = scraper.get(url)
-			if r.status_code == 200:
-				with open(outfile, 'wb') as f:
-					f.write(r.content)
-			else:
-				print("Encountered Error {} when downloading.".format(e.code))
+			for _ in range(0,10):
+				try:
+					r = scraper.get(url)
+					if r.status_code == 200:
+						with open(outfile, 'wb') as f:
+							f.write(r.content)
+				except:
+					print("Encountered an error when downloading. Retrying...")
```

Comment:
Maybe it would be better to keep the old error, but only print it when bailing out after the last retry?